README.md

```diff
@@ -30,13 +30,14 @@ To do this, you'll need to visit that page, and click on the 3 dots, and select
 
 ##### Using npm:
 Add the following to your `.cursor/mcp.json` or `claude_desktop_config.json` (MacOS: `~/Library/Application\ Support/Claude/claude_desktop_config.json`)
+Note that the version is pinned.
 
 ```javascript
 {
   "mcpServers": {
     "notionApi": {
       "command": "npx",
-      "args": ["-y", "@notionhq/notion-mcp-server"],
+      "args": ["-y", "@notionhq/notion-mcp-server@1.8.1"],
```

Comment:
Thanks for the suggestion. However, I don't think this should be the default as we want users to automatically use the latest and greatest. The same risk can be said about pinned versions that later have discovered vulnerabilities. This can create a situation where there are lot users who depend on legacy versions and make it much harder to deprecate problematic features.